content/warnings/invalid-aria-prop.md

```diff
@@ -4,8 +4,8 @@ layout: single
 permalink: warnings/invalid-aria-prop.html
 ---
 
-The invalid-aria-prop warning will fire if you attempt to render a DOM element with an aria-* prop that does not exist in the Web Accessibility Initiative (WAI) Accessible Rich Internet Application (ARIA) [specification](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
+O aviso invalid-aria-prop irá disparar caso você tente renderizar um elemento do DOM com uma propriedade aria-* que não existe na Iniciativa para a Acessibilidade da Web (WAI) Aplicações Internet Ricas Acessiveis (ARIA) [especificação](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
```

Comment:
let's avoid translating WAI and ARIA

Reply:
Fixed

Reply:
@cezaraugusto I changed the anchor order to make more sense. Is this correct?

Reply:
sounds good 👍